foundation/foundation-mda/src/main/resources/templates/deployment/airflow/v2/airflow.values-dev.yaml.vm

```diff
@@ -23,7 +23,7 @@ aissemble-airflow-chart:
         # (such as the current project version) this attribute can be removed:
         worker_container_tag: latest
 
-    defaultAirflowRepository: boozallen/${projectName}-airflow-docker
+    defaultAirflowRepository: ${projectName}-airflow-docker
```

Comment:
C: Shouldn't this be:
```suggestion
    defaultAirflowRepository: ${dockerProjectRepositoryUrl}/${projectName}-airflow-docker
```

Reply:
Ah. I see now this is dev and so we're trying to strip the remote repo to force use of a local image.  This isn't necessary given that we build images with their full registry URL locally.  We can just drop this override entirely and use the setting from the base `values.yaml` even locally, I think.

Reply:
Ah ok, I can remove it from the dev files I added it to. Also based on the deployment.yaml files, it looks like they all want a similar format to: `{{ .Values.deployment.image.repo }}{{ .Values.deployment.image.name }}:{{ .Values.deployment.image.tag | default .Chart.AppVersion}}`

So for guidance, the `dockerProjectRepositoryUrl` should contain the "/" (just FYI since I was you added it in your suggestion)

Reply:
Downgraded this to Substantive btw, given the point about Tilltfile made in the other comment.

Reply:
FYI leaving the repo blanked out in the values-dev files. Want to make sure we test that thoroughly instead of just adding that in here.